cdbot/cogs/cyber.py

```diff
@@ -169,7 +169,7 @@ async def flag(
             # Generates random, but unique and identical per challenge, base 64 "flag"
             content = (
                 "The flag is:"
-                f"||{await generatebase64(ord(base[0]) + level_num + challenge_num)}||"
+                f"https://game.joincyberdiscovery.com/assets/videos/cheating_message.mp4?version=4.2.0"
```

Comment:
Setting aside that this wont work, this doesn't really do what was discussed in #481

Reply:
Oh sorry, I didn't see 481 before this, my bad